packages/edit-post/src/style.scss

```diff
@@ -61,8 +61,11 @@ body.block-editor-page {
 	@include wp-admin-reset( ".block-editor" );
 }
 
-.block-editor,
-// The modals are shown outside the .block-editor wrapper, they need these styles
+// Target the editor UI excluding the metaboxes and custom fields areas.
+.editor-styles-wrapper,
```

Comment:
Should we replace this one with `.edit-post-visual-editor` instead? 
What about the text editor `.edit-post-text-editor`?
I also noticed that the notices are shown outside that div should we add the reset for them or is it unnecessary:

```
.edit-post-layout__content > .components-notice-list,
.edit-post-layout__content > .components-snackbar-list
```

Reply:
What about the top bar as well?

Reply:
> Should we replace this one with .edit-post-visual-editor instead?

Aren't they equivalent? If you mean for "semantic" reasons, no objections!

> What about the text editor .edit-post-text-editor?

Yup, looks like it's needed for the permalinks UI there. Seems to me that's the only case of form controls within the text-editor area. Will add.

> I also noticed that the notices ar

Yep, I did notice the notices. Not sure: is Gutenberg going to use form controls within the notices? It may need the `box-sizing: border-box;` part.

> What about the top bar as well?

Same: will there be form controls within the top bar? I think the top bar needs only the `box-sizing: border-box;` part.

Reply:
> Aren't they equivalent? If you mean for "semantic" reasons, no objections!

yes, mosty for semantic but also to prevent bugs as `editor-styles-wrapper` can be used in several places to say "apply the editor styles here" (HTML prevew, block previews...)